examples/federation/README.adoc

```diff
@@ -0,0 +1,222 @@
+== Coherence Operator Federation Example
+
+This simple example demostrates the Coherence federation feature.  It shows how to deploy two Coherence clusters that federating data between them using the Coherence Operator. The Coherence federation feature requires Coherence Grid Edition.
+
+You can find the source code in the https://github.com/oracle/coherence-operator/tree/master/examples/federation[Operator GitHub Repo].
+
+* <<pre,Prerequisites>>
+* <<build, Build the Example>>
+* <<example, Run the Example>>
+* <<cleanup, Cleaning Up>>
+
+[#pre]
+== Prerequisites
+
+Following the instructions in https://github.com/oracle/coherence-operator/tree/master/examples/deployment[Coherence Operator Deployment Example] for all the prerequisites and installing the Coherence Operator. 
```

Comment:
We shouldn't make people have to look at a different example to be able to use this one.
There just needs to be an Operator running, the instructions for that are simple:
```
kubectl apply -f https://github.com/oracle/coherence-operator/releases/download/v3.1.5/coherence-operator.yaml
```